src/app/modules/browse/browse-controller.js

```diff
@@ -83,23 +83,38 @@
                 params.bookOrBooks = _localize('book');
             }
 
+            var isFiltered = false;
+
             if (count === 0) {
-                message = _localize('There are no books that match your search for {shelf} {language} books', params);
+                $scope.resultsMessage = _localize('There are no books that match your search', params);
```

Comment:
Grant, doesn't this change the following one remove a feature of the program to tell people what the search is that failed? Is there some reason that you'd want to do that?